company-profiles/hopper.md

```diff
@@ -0,0 +1,43 @@
+# Hopper
+
+## Company blurb
+
+Hopper is a company developing a mobile app for booking flights and hotels. It predicts and analyzes airfare and accommodation helping users to plan, book, and pay trips.
+
+## Company size
+
+Hopper has 437 employees around the globe.
+
+## Remote status
+
+Some employees work full-time from our offices and some work
+remotely. There are a variety of hybrid and fully remote roles at Hopper.
```

Comment:
There is an odd line break here

Reply:
@dougaitken I fixed the line break.

Reply:
It is still there. The line break between `some work` and `remotely`